### PR TITLE
Flash briefing

### PR DESCRIPTION
**Pull request by galactoise**
@davidwphillips Small PR for the FlashBriefing POJO, and also disabling a test that shouldn't run during mvn install

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -108,6 +108,12 @@
       <classifier>jdk15</classifier>
       <scope>test</scope>
     </dependency>
+    
+    <dependency>
+	    <groupId>com.google.code.findbugs</groupId>
+	    <artifactId>findbugs</artifactId>
+	    <version>3.0.1</version>
+	</dependency>
```
Comment (davidwphillips) [inline pom.xml:116]: We should let this demon die. It only haunts our souls

File: voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/alexa/FlashBriefingResponse.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+@SuppressFBWarnings
+public class FlashBriefingResponse {
+  
+  public static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'.0Z'");
+  
+  private String uid;
+  private String titleText;
+  private String mainText;
+  @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd'T'HH:mm:ss'.0Z'", timezone="UTC")
+  private Date updateDate;
```
Comment (davidwphillips) [inline voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/alexa/FlashBriefingResponse.java:19]: Is it possible to use a better time-related class? I thought we were supposed to be moving away from Date. Something like ZonedDateTime:
http://stackoverflow.com/questions/31627992/spring-data-jpa-zoneddatetime-format-for-json-serialization

File: voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/alexa/FlashBriefingResponse.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+
+@SuppressFBWarnings
+public class FlashBriefingResponse {
+  
+  public static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'.0Z'");
```
Comment (davidwphillips) [inline voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/alexa/FlashBriefingResponse.java:13]: Didn't we want a format like this instead?
@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ")

Comment (galactoise) [reply voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/alexa/FlashBriefingResponse.java:13]: Flash Briefing API is very particular about its date format.

File: pom.xml (hunk 1 above)
Comment (galactoise) [reply pom.xml:116]: It does catch some things. I wish we could just configure it to not be annoying about trivial issues.

**Pull request merged**